oval_graph/_builder_html_graph.py

```diff
@@ -1,3 +1,7 @@
+'''
+    This file contains class for building HTML with graph
```

Comment:
The file contains a class called `BuilderHtmlGraph` - the name already indicates that it builds a HTML with the graph.